disasm.py

```diff
@@ -148,76 +148,106 @@ def addComments(script, actionLines):
                 isFloatMask = 0xF000000
                 extractMSBMask = 0xFF000000
                 extractDecimalIDMask = 0xFFFFFF
-                if parameter & isBitOrBasicMask: # if basic or bit
+                if isinstance(parameter, int) and parameter & isBitOrBasicMask: # if basic or bit
```

Comment:
Isn't parameter & isBitOrBasicMask an integer? I don't think you should be treating an int as a bool in python (as opposed to C)

Reply:
Apparently bool is a subclass of int in Python, so the conditional already does a downcast and it functions in the way that I want. I still changed it to "!= 0x0" for piece of mind, though